routes/categoryRouter.js

```diff
@@ -0,0 +1,7 @@
+const express = require("express");
+const router = express.Router();
+const categoryController = require("../controllers/categoryController");
+
+router.get("/:categoryId", categoryController.categoryInfo);
```

Comment:
```suggestion
router.get("/products?categoryId=1", productController.getProducList);
```
- 이게 RESTful API의 정석 방식입니다.
- 프론트 로직 변경할 여유 있으면, 수정하시는게 좋고, 어려우면 일단 유지 후 리팩토링